src/components/login/login.js

```diff
@@ -146,44 +145,41 @@ class Login extends React.Component {
   }
 
   render() {
+    const networkList = [{ label: this.props.t('Choose Network'), disabled: true }, ...this.networks];
     return (this.props.account.loading ?
       <span></span> :
       <Box className={styles.wrapper}>
         <section className={`${styles.login} ${styles[this.state.passInputState]}`}>
           <section className={styles.table}>
             <header>
-              <a className={styles.forwardToLink} href='https://lisk.io' target='_blank' rel='noopener noreferrer'>
-                <span className={styles.label}>{this.props.t('Go to lisk.io')}</span>
-                <FontIcon className={styles.icon}>arrow-right</FontIcon>
-              </a>
+              {this.showNetworkOptions()
+                ? <div>
```

Comment:
I think the div is not needed if it's not adding extra styles

Reply:
The div is wrapping 2 components, they need some kind of wrapper